dockerfiles/base/scripts/base/commands/cmd_destroy.sh

```diff
@@ -78,3 +66,12 @@ cmd_destroy() {
   fi
 }
 
+directory_is_empty() {
```

Comment:
They had different syntax of what to check.  So I keep both as `directory_is_empty` and then add a qualifier for the one in config now.